topics/01_General-and-Getting-Started/~Getting_Started.md

```diff
@@ -8,25 +8,237 @@
+Ignite UI&trade; is an advanced HTML5+ toolset that helps you create stunning, modern Web apps. Building on jQuery and jQuery UI, it primarily consists of feature rich, high-performing UI controls/widgets such as all kinds of charts, data visualization maps, (hierarchical, editable) data grids, pivot grids, enhanced editors (combo box, masked editors, HTML editor, date picker, to name a few), flexible data source connectors, and a whole lot more.
 
-## Jump Right In
-If you'd like to dive right in by working with the controls directly, the [Page Designer allows you to work with each control in a WYSIWYG environment](http://designer.igniteui.com/). The designer comes is complete with sample data that makes it easy to get started without having to go through any sort of setup process.
+Ignite UI comes in two vesions: 
+-   Open Source - a free version that contains a subset of the complete toolset. Grids and Data Visualization controls are excluded. For more information checkout the [Ignite UI OSS](https://github.com/IgniteUI/ignite-ui) project on GitHub&trade;.
+-   Full - a paid version that contains the complete toolset.
 
-## Building Your First Page
-As you begin to build your first page you can follow the [Ignite UI Getting Started tutorial](http://www.igniteui.com/getting-started) which demonstrates how to get setup and  shows you which files to reference on the page.
+# <a id="download"></a>Download and Install
 
-## Deploying Ignite UI
-Once your page is built, then you'll want to review the [deployment guide](Deployment-Guide.html) which helps you get better acquainted with not only all the individual files involved in using Ignite UI, but also where and how to deploy them depending on how your application is configured. 
\ No newline at end of file
+You can download the Ignite UI from its [download page](http://www.infragistics.com/products/ignite-ui/download). 
+Depending on your operating system you have several options to install the product, both trial and licensed.
+
+For Windows:
+
+- **[Platform Installer](https://www.infragistics.com/my-account/keys-and-downloads)**: Platform Installer is a Windows only online installer that gives you a wizard like experience to manage (install, update, uninstall) Ignite UI and all other Infragistics Developer tools their help and samples.
+
+- **[Download full product](http://www.infragistics.com/products/ignite-ui/download)**: 
+A Windows only offline MSI installer containing the product, help and samples.
+
+For MacOs, Linux, Unix etc.
+
+- **[Download No Installer Bundle](http://www.infragistics.com/products/ignite-ui/download)**: 
+A Windows only offline MSI installer containing the product and samples.
+
+# <a id="hosting"></a>Host Ignite UI in your project
+
+You have several options to host Ignite UI in your project:
```

Comment:
Order the bullets as per the section structure below in the topic:
- Use Package Manager like NPM, JSPM, NuGet
- Copy the [JavaScript and CSS files](Adding_the_Required_Resources_for_NetAdvantage_for_jQuery.html) from the Ignite UI installation directory into your project 
- Use [Infragistics Content Delivery Network (CDN)](#cdn)